perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -242,11 +194,12 @@
 DEFAULT = 'default'
 
 # This accounts for the overhead of running RemoteCommand() on a VM.
-MULTISTREAM_DELAY_PER_VM = 5.0
-# We wait this many seconds for each stream. Note that this is
-# multiplied by the number of streams per VM, not the total number of
-# streams.
-MULTISTREAM_DELAY_PER_STREAM = 0.1
+MULTISTREAM_DELAY_PER_VM = 10.0 * units.second
+# We wait this long for each stream. Note that this is multiplied by
+# the number of streams per VM, not the total number of streams.
+MULTISTREAM_DELAY_PER_STREAM = 15.0 * units.second
```

Comment:
We probably need a better way to handle this. Say we have 200 streams and 10 vms, we will be waiting 8,9 hours before this test is run.